java/spatial-operations/README.md

```diff
@@ -1,22 +1,41 @@
-# Spatial Operations
-Perform geometry-on-geometry spatial operations by using `GeometryEngine`. Two input `Polygons` are added as `Graphics` to a `GraphicsOverlay` and displayed in a `MapView`. 
+# Perform spatial operations
 
-![Spatial Operations App](spatial-operations.png)
+Find the union, intersection, or difference of two geometries.
+
+![Image of perform spatial operations](spatial-operations.png)
+
+## Use case
+
+The different spatial operations (union, difference, symmetric difference, and intersection) can be used for a variety of spatial analyses. For example, government authorities may use the intersect operation to determine whether a proposed road cuts through a restricted piece of land such as a nature reserve or a private property.
+When these operations are chained together, they become even more powerful. An analysis of food deserts within an urban area might begin by union-ing service areas of grocery stores, farmer's markets, and food co-ops. Taking the difference between this single geometry of all services areas and that of a polygon delineating a neighborhood would reveal the areas within that neighborhood where access to healthy, whole foods may not exist.
 
 ## How to use the sample
-Using the options menu, different spatial operations can be performed, the result of which is shown as a third `Graphic`, in red, in the `MapView`. Additionally, the `union` method is used to create an initial geometry with which to set the `Viewpoint` of the `MapView`.
+
+The sample provides an option to select a spatial operation. When an operation is selected, the resulting geometry is shown in red. The 'reset operation' button undoes the action and allow selecting a different operation.
```

Comment:
I would specify that the options are available in the overflow menu. The first sentence here is very strangely worded anyway.

Reply:
There is also no reset operation button, and there is a typo here ("allow" -> "allows"). There is a "no operation" option in the menu, but there is nothing blocking the user from selecting a new option directly.